### DISCUSSION TITLE
Demo non funzionante

### DISCUSSION THREAD

**Discussion by emiliana32** [iisgiua/giuaschool#345] [Richieste di aiuto]
Salve,
ho scoperto di recente questo progetto. Non sono riuscita a provarlo poiché la demo non funziona. Come posso fare?
Grazie

**Comment by trinko** [comment]
Puoi scaricare il software su un tuo PC e installarlo lì. 
Vedi la documentazione:
[Installazione tramite Docker](https://iisgiua.github.io/giuaschool-docs/install-docker.html)

**Reply by emiliana32** [reply]
Grazie della risposta. Certo, ma allora la demo andrebbe rimossa se non funzionante.

**Reply by manuelbarberi** [reply]
Sto tendando di riaprire la demo... per richieste scriva alla mail giuaschoolregistro@gmail.com

**Reply by emiliana32** [reply]
Grazie

**Comment by Davide031** [comment]
Salve se vuole provare una demo del registro mi scriva su fcoco953@gmail.com

**Comment by ghost** [comment]
@emiliana32 Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
